.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: Python package
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [2.7.18]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt-get update && sudo apt-get install virtualenv gettext
+    - name: Run setup
+      run: |
+        CREDENTIALS=(
```

Comment:
Nevolal by som to `CREDENTIALS`, kedze to nie su credentials. Skor `CONFIGURATION` alebo tak nejak.